src/client/lobby/room-creator.js

```diff
@@ -5,33 +5,33 @@ import config from '../config/config';
 import style from './lobby.scss';
 
 export default class RoomCreator extends Component {
-
-    onUsernameClick(){
-        var username = document.getElementById('user-name').value;
-        this.props.setUsername(username);
-    }
-
-    onRoomnameClick(){
-        var roomname = document.getElementById('room-name').value;
-        this.props.setRoomname(roomname);
-    }
-
     render() {
         return (
             <div className="col-lg-4 text-center">
                 <h2>Create Room & Set Username</h2>
                 <div className="col-lg-12 list-group">
                     <div className={`row ${style.space}`}>
                         <input id="user-name" type="text" placeholder="Please type your new username"></input>
-                        <button id="user-name-button" className="btn btn-success" onClick={this.onUsernameClick.bind(this)}>Set Username</button>
+                        <button id="user-name-button" className="btn btn-success" onClick={onUsernameClick.bind(this)}>Set Username</button>
                     </div>
                     <br></br>
                     <div className="row">
                         <input id="room-name" type="text" placeholder="Please type new room name"></input>
-                        <button id="room-name-button" className="btn btn-success" onClick={this.onRoomnameClick.bind(this)}>Create Room</button>
+                        <button id="room-name-button" className="btn btn-success" onClick={onRoomnameClick.bind(this)}>Create Room</button>
                     </div>
                 </div>
             </div>
         );
     }
 };
+
+function onUsernameClick() {
+    var username = document.getElementById('user-name').value;
```

Comment:
There has to be a better way that this to get the content of a an input field than ids...can you have the input field in as a public variable on the class or something?